### PR TITLE
zapier-platform-core: Add methods related to tests

### PR DESCRIPTION
**Pull request by janaagaard75**
Please fill in this template.

- [*] Use a meaningful title for the pull request. Include the name of the package modified.
- [*] Test the change in your own code. (Compile and run.)
- [*] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [*] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [*] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not provide its own types, and you can not add them.
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [*] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/zapier/zapier-platform-core/blob/master/src/index.js
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/zapier-platform-core/index.d.ts [hunk 1]
```diff
@@ -8,6 +8,9 @@
 
 import * as http from "http";
 
+export const createAppHandler: Function;
```
Comment (andy-ms) [inline types/zapier-platform-core/index.d.ts]: Do you know what the parameters and return type should be?

File: types/zapier-platform-core/zapier-platform-core-tests.ts [hunk 2]
```diff
@@ -91,3 +92,10 @@ const authentication: OAuth2Authentication = {
         return true;
     }
 };
+
+zapier.version;
+
+const testCreateAppTester = async () => {
+    const zapierApp = {};
+    const appTester = zapier.createAppTester(zapierApp);
```
Comment (andy-ms) [inline types/zapier-platform-core/zapier-platform-core-tests.ts]: You probably meant to `await` this.

**Review by ghost — CHANGES_REQUESTED** [review]

File: types/zapier-platform-core/index.d.ts (hunk 1 above)
Comment (janaagaard75) [reply types/zapier-platform-core/index.d.ts]: No, I didn't dig into this, but still thought that it was better to add it as a generic function. Would you prefer that I remove it?

**Review by bradleyayers — APPROVED** [review]

File: types/zapier-platform-core/index.d.ts (hunk 1 above)
Comment (andy-ms) [reply types/zapier-platform-core/index.d.ts]: The `Function` type fails the lint, so replace it with `(...args: any[]) => any` at least.

Comment (janaagaard75) [reply types/zapier-platform-core/index.d.ts]: Thanks a lot for pointing out the solution. You're right that it was a bit lazy to just use a generic function, so I tried to specify them better, but must admit, that they're only slightly more specific in the updated pull request.

File: types/zapier-platform-core/zapier-platform-core-tests.ts (hunk 2 above)
Comment (janaagaard75) [reply types/zapier-platform-core/zapier-platform-core-tests.ts]: Right you are. It has been added.

**Comment by andy-ms** [comment]
Remaining lint errors look pretty self-explanatory.

**Comment by janaagaard75** [comment]
Just wanted to say thanks for all the help with the pull request.

**Comment by andy-ms** [comment]
Thanks!

**Pull request merged**